hogwatch/server/static/js/main.js

```diff
@@ -104,9 +104,12 @@ function seriesBlueprint(mode){
 var defaultInterfaces=['all'];
 
 function getSocketURL(interface_list,mode){
-    var host=location.host;    
```

Comment:
maybe something like this would be better
```js
var socketProtocol = location.protocol === 'https:' ? 'wss://' : 'ws://';
var socketURL=socketProtocol+host+'/websocket/'+interface_list.join('_')+'/'+mode;
```